app/backend/src/couchers/jobs/handlers.py

```diff
@@ -484,6 +487,38 @@ def send_reference_reminders(payload):
 send_reference_reminders.SCHEDULE = timedelta(hours=1)
 
 
+def send_host_request_reminders(payload):
+    with session_scope() as session:
+        requests = session.execute(
+            select(HostRequest)
+            .where(HostRequest.status == HostRequestStatus.pending)
+            .where(HostRequest.host_sent_request_reminders < HOST_REQUEST_MAX_REMINDERS)
+            .where(HostRequest.from_date > today_in_timezone(HostRequest.timezone))
```

Comment:
Use `HostRequest.start_time > func.now()` here: that one already takes care of the timezone (and will eventually care about the host's timezone.